tests/testthat/test-build-article.R

```diff
@@ -0,0 +1,20 @@
+context("build_article")
```

Comment:
Tests should live in a file with the same name as the `R/` file, so can you please place in `test-rmarkdown.R`